File: Develop/src/Books.java
public class Books {

    private String name;
    private String autor;
    private boolean disponible;

    public boolean getDisponible() {
        return disponible;
    }

    public void setDisponible(boolean disponible) {
        this.disponible = disponible;
    }


    public Books(String name,String autor,boolean disponible){
        this.name=name;
        this.autor= autor;
        this.disponible=disponible;

    }

    public String getName() {
        return name;
    }

    public String getAutor() {
        return autor;
    }
}

File: Books.py
class Books:
    def __init__(self, name, autor, disponible):
        self.name = name
        self.autor = autor
        self.disponible = disponible

    def getDisponible(self):
        return self.disponible

    def setDisponible(self, disponible):
        self.disponible = disponible

    def getName(self):
        return self.name

    def getAutor(self):
        return self.autor
